README.md

```diff
@@ -91,3 +91,27 @@ generated `mapping.txt` file manually before any stack traces received from that
   - `ndk (true|false)` the default value is `false`. You can set it to `true` if you want to upload symbols for your native code as well.
 
 [programmatic-access]: https://developer.yahoo.com/flurry/docs/api/code/apptoken/
+
+
+## Publishing new versions of this plugin
```

Comment:
I'm leaning towards adding the instructions to publish this plugin in a separate file.